crates/symbolicator-service/Cargo.toml

```diff
@@ -14,7 +14,7 @@ aws-config = { version = "1.0.1", features = ["behavior-version-latest"] }
 aws-credential-types = { version = "1.0.1", features = ["hardcoded-credentials"] }
 aws-sdk-s3 = "1.4.0"
 aws-types = "1.0.1"
-cadence = "1.0.0"
+cadence = "0.29.0"
```

Comment:
Why do we need to downgrade here? Because `statsdproxy` is still on the old version?